dot/core/messages_test.go

```diff
@@ -335,9 +315,8 @@ func TestServiceHandleTransactionMessage(t *testing.T) {
 					tt.mockBlockState.bestHeader.err)
 
 				if tt.mockBlockState.getRuntime != nil {
-					blockState.EXPECT().GetRuntime(gomock.Any()).Return(
-						tt.mockBlockState.getRuntime.runtime,
-						tt.mockBlockState.getRuntime.err)
+					blockState.EXPECT().GetBestBlockRuntime().Return(
+						tt.mockBlockState.getRuntime.runtime)
```

Comment:
supernit better cut it after the `.`

```suggestion
					blockState.EXPECT().GetBestBlockRuntime().
						Return(tt.mockBlockState.getRuntime.runtime)
```